extensions/stats/proxy_expr.h

```diff
@@ -0,0 +1,122 @@
+/* Copyright 2020 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// WASM_PROLOG
```

Comment:
Is this temporary or will it always be there?
Is this is temporary we should add a comment here.

Reply:
Temporary. This whole file needs to be exported from upstream. It's a verbatim copy.

Reply:
Need top push wasmsdk:v4 to get rid of this file at some point. It's identical to upstream.